tests/api/server_test.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from fastapi.testclient import TestClient
+from ff_fasttext_api.server import make_app
+from unittest.mock import MagicMock
+from bonn.extract import CategoryManager
+from ff_fasttext_api.healthcheck import Healthcheck
+
+@pytest.fixture(scope='module')
+def test_client():
+    # Create a mock CategoryManager object to simulate the FastText model
```

Comment:
This is useful, but an integration test that uses the sample data with the real category manager (bonn) would be good -- for instance, there is no test guaranteeing that our assumed return structure is what bonn would send (interface mismatch is a common pitfall for an overly encapsulated test suite, but equally, we can't depend on the entire environment - hence, a minimal sample fifu model is in the code base)